lib/daru.rb

```diff
@@ -26,19 +26,20 @@ module Daru
     10 => 31,
     11 => 30,
     12 => 31
-  }
+  }.freeze
 
-  SPLIT_TOKEN = ','
-  class << self
-    @@lazy_update = false
+  @lazy_update = false
```

Comment:
One of the main arguments I've seen against class instance variables is that when they are modified in an inherited class, they also affect the parent class.

Here's one of my go-to articles on class variables that goes into more depth:  http://www.railstips.org/blog/archives/2006/11/18/class-and-instance-variables-in-ruby/